python/ray/tests/test_client.py

```diff
@@ -386,6 +398,7 @@ def test_log(level, msg):
         assert ray.get(x) == "Foo"
         time.sleep(1)
         logs_with_id = [msg for msg in log_msgs if msg.find(x.id.hex()) >= 0]
+        print(logs_with_id)
```

Comment:
Is this intended?

Reply:
No, changed to print under assertion failure.